### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by thangzathang**

### CODE REVIEW

File: src/components/filterBar/FilterBar.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const FilterBar = (props) => {
       [e.target.name]: value,
     });
   };
-  console.log(filters);
+  // console.log(filters);
 
   return (
     <FilterContainer>
```
Comment (a-sh-dev) [inline src/components/filterBar/FilterBar.js:41]: I think the whole `FilterBar.js` and its folder have been deleted by Winny.. so you can discard this file and all the related imports.

She said she couldn't resolve the issue even with the help from Iryna and the others

**Review by a-sh-dev — COMMENTED** [review]
I hope it's not too much of a conflict to be resolved Thang, you can delete the `filterbar` and all its contents and related imports as it's been deleted

**Review by thangzathang — COMMENTED** [review]
New code checked

**Comment by a-sh-dev** [comment]
This branch seems to be behind 34 commits from the latest `main` Thang? 😱 →  https://github.com/DIYGrooming/client-frontend/tree/Pagination

**Pull request merged**